docs/rules/require-render-return.md

```diff
@@ -1,6 +1,8 @@
-# Enforce ES5 or ES6 class for returning value in render function (react/require-render-return)
+# Enforce ES5 or ES6 class for returning value in render function (`react/require-render-return`)
 
-💼 This rule is enabled in the following [configs](https://github.com/jsx-eslint/eslint-plugin-react#shareable-configurations): `all`, `recommended`.
+✅ This rule is enabled in the `recommended` [config](https://github.com/jsx-eslint/eslint-plugin-react/blob/master/README.md#shareable-configs).
```

Comment:
oh and actually, this is in the main readme, so why even have the blob/ part? can these links just be to the repo directly?